src/test/java/com/intuit/graphql/orchestrator/xtext/XtextScalarsTest.java

```diff
@@ -19,21 +9,11 @@
 
   @Test
   public void testDoesNotReturnSameInstance() {
-    assertThat(newBigDecimalType()).isNotSameAs(newBigDecimalType());
```

Comment:
just checking if this tests should be removed ?

Reply:
Did not mean to remove; that was just for testing